tests/sql/test_db_connector.py

```diff
@@ -66,9 +66,9 @@ def test_insert_versions(dbsession):
 
     inserted_versions = dbsession.query(Version)
     assert inserted_versions.count() == 3
-    for idx, _ in enumerate(inserted_versions):
-        assert inserted_versions[idx].technology == techno
-        assert inserted_versions[idx].version == versions[idx]
+    for idx, inserted_version in enumerate(inserted_versions):
+        assert inserted_version.technology == techno
+        assert inserted_version.version == versions[idx]
```

Comment:
The pythonic way to match item by item two lists of the same size is to use `list.zip(list)`.

In this case:

```suggestion
    for inserted_version, expected version in inserted_versions.zip(versions):
        assert inserted_version.technology == techno
        assert inserted_version.version == expected_version
```

But the main goal of this MR is to clean pylint output.